client/src/types.ts

```diff
@@ -12,10 +12,17 @@ export interface ChatData {
 }
 
 export interface ChatMsg {
+  // we should change name, color, and isUser to just mentorId
+  name: string;
+  color: string;
+  mentorId: string;
   isUser: boolean;
   text: string;
-  feedback?: Feedback;
-  feedbackId?: string;
+  feedback: Feedback;
+  feedbackId: string;
+  isFeedbackSendInProgress: boolean;
+  visibility: boolean;
+  clicked: boolean;
```

Comment:
whatever `clicked` might used for here it seems wrong from a data-modeling perspective. Do we need this?